.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-package-lock.json
\ No newline at end of file
+package-lock.json
+.env
```

Comment:
this means the .env file will never be committed, and will stay in your local setup only